css/properties/hyphenate-character.json

```diff
@@ -0,0 +1,69 @@
+{
+  "css": {
+    "properties": {
+      "hyphenate-character": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/hyphenate-character",
+          "spec_url": "https://drafts.csswg.org/css-text-4/#propdef-hyphenate-character",
+          "support": {
+            "chrome": {
+              "prefix": "-webkit-",
+              "version_added": "37"
```

Comment:
Finally, a chance to exercise my ancient browser research skills. Here's how I did it:

1. If you switch operating systems, you can go back to Chrome 15 or so on BrowserStack. It works there, which gets us in the range of Chromium releases which used WebKit (before Blink emerged). Range narrowed to Chrome 1-15.
2. I tested it in older Safaris to rough things out a little more. It shows up in Safari 5.1 (WebKit ~534.48). This puts us in the range of Chrome 1-12, based on WebKit version mapping (i.e., it wouldn't have shown up any later than the Chrome that ships WebKit 534.48 or greater).
3. I searched [webkit/webkit](https://github.com/webkit/webkit) to find the changset that added it: https://trac.webkit.org/changeset/61548/webkit. This dates the change to circa June 2010, narrowing things to Chrome 6-12.
4. A comment in [the associated WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=10228) mentions a Chromium performance regression. This suggests that Chrome was updating WebKit versions pretty promptly. Based on the date, I think we can safely assume Chrome 6. But just to be sure, the we can compare [WebKit tags](https://trac.webkit.org/browser/webkit/tags) to the revision number, which places this around WebKit 534.1, which also aligns with Chrome 6.

```suggestion
              "version_added": "6"
```

Updating the rest of the values is an exercise in version mapping.